DotNetHowToAutocomplete/DotNetHowToAutocomplete/Web.config

```diff
@@ -0,0 +1,65 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+  For more information on how to configure your ASP.NET application, please visit
+  https://go.microsoft.com/fwlink/?LinkId=301880
+  -->
+<configuration>
+  <appSettings>
+    <add key="SearchServiceName" value="azs-playground" />
+    <add key="SearchServiceApiKey" value="19DCAA183875A7D5E4DFF47D3CDB498B" />
```

Comment:
Is that a query key or an admin key? If it's an admin key, you need to remove it from here and change it ASAP.

If it's a query key, who's on the hook to maintain the indexes in the azs-playground service? Will it always be available for people to use?

We avoid these thorny problems in other samples by letting customers provide their own search service name and api-key in config, and then include a utility to load data.

Reply:
This is the same key used in the NYC jobs sample: https://github.com/Azure-Samples/search-dotnet-asp-net-mvc-jobs/blob/master/NYCJobsWeb/Web.config

Reply:
I was looking for that but couldn't find it. Thanks for confirming.